moveit_core/constraint_samplers/src/constraint_sampler_tools.cpp

```diff
@@ -37,10 +37,13 @@
 #include <moveit/constraint_samplers/constraint_sampler_tools.h>
 #include <moveit/constraint_samplers/constraint_sampler_manager.h>
 
+rclcpp::Logger LOGGER_CONTRAINT_SAMPLERS_TOOLS = rclcpp::get_logger("moveit").get_child("constraint_samplers");
```

Comment:
Put in namespace. This could live in the .hpp or you can use a namespace block for `constraint_samplers` in the CPP. Using the namespace block would have the happy side effect of reducing the width of code.

Reply:
https://github.com/ros-planning/moveit2/pull/60/commits/a4c9c83a4e39c6e7aabe2ac6bfcfde57d47b5cf5